functions.php

```diff
@@ -80,6 +80,54 @@ function new_display_post_thumbnail_column($col, $id){
   }
 }
 
+// Instagram Feed
+function get_instagram_feed() {
+  $instagram_handle = IGV_get_option( '_igv_instagram_handle' ); 
+  
+  $feed = get_transient( 'instagram_feed' );
+  // Transient yis a piece of informtion that may be or not stored in 
+  // ifast memory instead of in the db. This data es expected to expire,
+  // or could expire at any time. 
+
+  // If feed doesn't exist
+  if ( empty($feed) ) {
+    $url = 'https://instagram.com/' . $instagram_handle . '/media/';
+
+    // Make API call to instagram
+    $response = wp_remote_get( $url );
+
+    if( !is_wp_error( $response ) ) {
+      $body = json_decode( $response['body'] );
+      $feed = $body->items;
+
+      // Slice first 5 elements
+      $feed = array_slice($feed, 0, 5);
+
+      // Remove unimportant data
+      foreach( $feed as &$feed_item ) {
+        unset(
```

Comment:
woah, i just realized i made this thinking this data was going to travel thru the wire ala meteor…
is definitely not needed